docs/apps/realtime-log-analytics.md

```diff
@@ -3,38 +3,32 @@ sidebar_position: 4
 title: Realtime Log Analytics
 ---
 
-Realtime Log Analytics integrating logs from Fastly with Macrometa GDN to monitor the status and activity of Stream workers and Query workers. Fastly provides data about HTTP latency, response count, response size, and unique visitor traffic.
-
+This demo shows how to integrate real-time log analytics with Macrometa GDN to monitor the status and activity of Stream workers and Query workers.
 
 ## Setup
 
 | **Federation**                                        | **Email**                              | **Passsword** | **Dashboard**|
 | ----------------------------------------------------- | -------------------------------------- | ------------- |--------------|
-| [Global Data Network](https://gdn.paas.macrometa.io/) | demo-fastly-realtime-logs@macrometa.io | `xxxxxxxx`    | [Dashboard](https://macrometacorp.github.io/demo-fastly-jsc8-realtime-logs) |
-| [Fastly Account](https://manage.fastly.com)           | demo@macrometa.com                     | --            | -- |
-
-
+| [GDN](https://gdn.paas.macrometa.io/) | demo-realtime-logs@macrometa.io | `xxxxxxxx`    | [Dashboard](https://macrometacorp.github.io/demo-jsc8-realtime-logs) |
 
 ## Solution
```

Comment:
This is the wrong heading. What follows are instructions or a procedure. Try replacing this with something more descriptive and adding a short introduction explaining to the user what they are going to do next. Provide context.